build.gradle

```diff
@@ -79,8 +79,8 @@ subprojects { subproject ->
 	}
 
 	// ensure JDK 6 compatibility
```

Comment:
This isn't true any more :smile: 
Or change it to `8` or just remove altogether!